docs/concepts/services/fetch.md

```diff
@@ -0,0 +1,236 @@
+    }
+    fn change(&mut self, _: Self::Properties) -> bool {
+        false
+    }
+    fn update(&mut self, msg: Self::Message) -> bool {
+        match msg {
+            GetLocation => {
+                // 1. build the request
+                let request = Request::get("http://api.open-notify.org/iss-now.json")
+                    .body(Nothing)
+                    .expect("Could not build request.");
+                // 2. construct a callback
+                let callback = self.link.callback(|response: Response<Json<Result<ISS, anyhow::Error>>>| {
+                    // split up the response into data about the request's status and the returned
+                    // data from the request
+                    let (meta, Json(Ok(data))) = response.into_parts();
+                    // the condition `data.message == "success" is specific to this API which
+                    // returns a JSON object with a "message" key – not all APIs will do this! 
+                    if meta.status.is_success() && data.message == "success" {
+                        Self::Message::ReceiveLocation(match data {
+                            Ok(d) => d,
+                            Err(e) => Self::Message::FetchError(e.to_string())
+                        })
+                    } else {
+                        Self::Message::FetchError(data.message)
+                    }
+                });
+                // 3. pass the request and callback to the fetch service 
+                FetchService::fetch(request, callback);
+                self.fetching = true;
+                // we want to redraw so that the page displays a 'fetching...' message to the user 
+                // so return 'true'
+                true
+            }
+            ReceiveLocation(location) => {
+                self.iss = Some(location);
+                self.fetching = false;
+                // we want to redraw so that the page displays the location of the ISS instead of 
+                // 'fetching...'
+                true
+            }
+            FetchError(error) => {
+                self.error = error;
+                // redraw to show the error
+                true
+            }
+            _ => false
+        }
+    }
+    fn view(&self) -> Html {
+        html! {
+            <>
+                {self.is_fetching()}
+                {self.view_iss_location()}
+                {
+                    if let Some(error) = self.error {
+                        <p>{error.clone()}</p>
+                    }
+                }
+            </>
+        }
+    }
+}
+```
+
+## Debugging the `FetchService`
+
+Most browsers' developer tools have a "network" pane which can be used to inspect and view requests 
+browsers have made, including data such as request headers and the contents of responses. This can 
+be a useful way to gain an insight into what requests are being made, the data being sent to the 
+server as well as the return data from the server.
```

Comment:
I feel like this should be reworded:
```suggestion
Most browsers' developer tools have a "network" panel which can be used to inspect HTTP requests. 
This can be used to gain insight into what requests are being made, the data being sent to the 
server, as well as the response.
```